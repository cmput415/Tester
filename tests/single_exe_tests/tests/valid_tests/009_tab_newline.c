
int main() {

    printf("\tabc\ndef\tehg\njki *END*");
    
    return 0;
}

// CHECK_FILE:./out-stream/009_tab_newline.out