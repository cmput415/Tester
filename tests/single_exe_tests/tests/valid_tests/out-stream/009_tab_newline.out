	abc
def	ehg
jki *END*